src/new-client/src/App/LiveRates/Tile/HistoricalGraph/HistoricalGraph.tsx

```diff
@@ -97,3 +97,59 @@ export const HistoricalGraph: React.FC<HistoricalGraphProps> = ({
     </LineChartWrapper>
   )
 }
+
+export const HistoricalGraphComponent: React.FC<{
```

Comment:
Similar to above comment on the Tile Header..

`HistoricalGraph` should be able to use the `HistoricalGraphComponent` to reduce the duplication.

Reply:
[fixed here](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/1970/commits/bebfc79ca887b2478f776dfaba36f2ce1f214d5c)